pkg/providers/launchtemplate/launchtemplate.go

```diff
@@ -89,7 +89,7 @@ type DefaultProvider struct {
 }
 
 func NewProvider(ctx context.Context, cache *cache.Cache, ec2api ec2iface.EC2API, eksapi eksiface.EKSAPI, amiFamily *amifamily.Resolver,
```

Comment:
Looked like you made this change for the other providers as well.
```suggestion
func NewDefaultProvider(ctx context.Context, cache *cache.Cache, ec2api ec2iface.EC2API, eksapi eksiface.EKSAPI, amiFamily *amifamily.Resolver,
```